app.py

```diff
@@ -74,6 +74,14 @@ def get_random_game_from_users_collection() -> Union[str, Response]:
         playstyle = None
     else:
         playstyle = args.get("playstyle")
+    if args.get("mincomplexity") == "":
```

Comment:
nit: a lot of these conditionals are checking to see if the provided value is an empty string, and then setting a local variable to `None` if the provided value is an empty string. I think we can clean up these conditionals by using an inline conditional like

`mincomplexity = args.get("mincomplexity") or None`

which I think should set `mincomplexity` to either the value of the request attribute or `None` in the case it is an empty string. The following is an example of what it looks like in a python3 REPL (but we should definitely test this if we go with this approach!)

```python
In [7]: a = {"foo": "bar", "baz": ""}

In [8]: print(a.get("foo") or None)
bar

In [9]: print(a.get("baz") or None)
None
```

sidenote: this is one of the things that flask-forms can help with a bunch in terms of setting sane defaults for parameters that aren't assigned a value from the request data,

Reply:
Ohhhhh I see interesting.  While I'm going to try to use flask-forms for this, I did not think of that simple 1 line conditional.